src/components/SuggestedValidators/ExpectedReturns.js

```diff
@@ -23,9 +28,22 @@ const ExpectedReturns = (props: ExpectedReturnsProps) => {
 				<Text fontSize='md' mt={4}>
 					Expected Returns
 				</Text>
-				<Text fontSize='2xl' fontWeight='medium'>
-					{returns} {props.currency}
+				<Text mb={4} fontSize='2xl' fontWeight='medium'>
+					<CountUp
+						end={returns}
+						decimals={3}
+						suffix={props.currency}
+					/>
 				</Text>
+				{props.button &&
+				<Link
+					href='/#/suggested-validators'
```

Comment:
Use hash router's way instead of this.

Useful resource: https://reacttraining.com/react-router/web/api/HashRouter

Reply:
Changed to hash router's link.